src/Input.js

```diff
@@ -3,8 +3,32 @@ import styled from "styled-components";
 import PropTypes from "prop-types";
 
 const Input = ({ onChange, value, label, type }) => {
+
+  const calculateOffset = () => {
+    const num = Array.from(String(value), Number);
+    let sum = 0;
+    let offsetVals = {
+      0: 14.27,
+      1: 16.39,
+      2: 18.85,
+      3: 19,
+      4: 20.02,
+      5: 19.05,
+      6: 18.25,
+      7: 16.03,
+      8: 18.58,
+      9: 18.26,
+    }
+    num.map((ea) => {
```

Comment:
You could use {`Array.prototype.reduce()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) here instead of `map`. Typically `map` is use to create a new, modified array from an existing one (e.g. take an array of JS objects and return an array of JSX components). `reduce` is designed to accumulate values from an array.